Sources/armory/network/WebSocket.hx

```diff
@@ -190,7 +190,9 @@ class WebSocket extends WebSocketCommon {
 	}
 
 	inline private function parseUrl(url)
-	{
+	{	
+		/** TO DO - FIND OUT WHAT IS BREAKING REGEX IN THE NEW PCRE2 FOR HL C 
```

Comment:
See https://github.com/Kode/Kha/pull/1478.

It's still a good idea to merge this I believe, depending on how long it takes for armory3d/Kha to be updated.